databricks/koalas/utils.py

```diff
@@ -288,6 +289,32 @@ def default_session(conf=None):
     return builder.getOrCreate()
```

Comment:
@ueshin, just out of curiosity, why don't we just enable it here?

Reply:
I mean, 

```python
session.conf.set(session.conf.get('spark.sql.execution.arrow.enabled', True))
```

So, if users disable it explicitly, just disable. Otherwise, enable it by default.

Reply:
That works too, but I didn't want to make unexpected behavior changes out of Koalas. Also we can focus on the Arrow enabled behavior. WDYT?